src/app/app-routing.module.ts

```diff
@@ -7,6 +7,10 @@ const routes: Routes = [
     path: '',
     loadChildren: () => import('./layout/layout.module').then(module => module.LayoutModule)
   },
+  {
+    path: 'sign-in',
+    loadChildren: () => import('./sign-in/sign-in.module').then(module => module.SignInModule)
```

Comment:
name of the lazy-loaded  module folder should start with `+` sign

Reply:
fixed